src/components/WearablePreview/WearablePreview.tsx

```diff
@@ -223,11 +222,10 @@ export class WearablePreview extends React.PureComponent<WearablePreviewProps> {
   }
 
   getOptions = () => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { dev, ...rest } = this.props
 
-    const options: PreviewOptions = {
```

Comment:
No need to send the `env` via postMessage, since the env vars are now controlled via `@dcl/ui-env`